python/foglamp/common/storage_client/payload_builder.py

```diff
@@ -90,8 +101,153 @@ def is_json(myjson):
         return True
 
     @classmethod
-    def ALIAS(cls, *args):
-        raise NotImplementedError("To be implemented")
+    def add_clause_to_select(cls, clause, qp_list, col, clause_value):
+        for i, item in enumerate(qp_list):
+            if isinstance(item, str):
+                if item == col:
+                    with_clause = OrderedDict()
+                    with_clause['column'] = item
+                    with_clause[clause] = clause_value
+                    qp_list[i] = with_clause
+            if isinstance(item, dict):
+                if 'json' in qp_list[i] and qp_list[i]['json']['column'] == col:
+                    qp_list[i][clause] = clause_value
+                elif 'column' in qp_list[i] and qp_list[i]['column'] == col:
+                    qp_list[i][clause] = clause_value
+
+    @classmethod
+    def add_clause_to_aggregate(cls, clause, qp_list, col, opr, clause_value):
+        if isinstance(qp_list, dict):
+            if 'json' in qp_list and qp_list['json']['column'] == col:
+                qp_list[clause] = clause_value
+            elif qp_list['column'] == col and qp_list['operation'] == opr:
+                qp_list[clause] = clause_value
+
+        if isinstance(qp_list, list):
+            for i, item in enumerate(qp_list):
+                if isinstance(item, dict):
+                    if 'json' in qp_list[i] and qp_list[i]['json']['column'] == col:
+                        qp_list[i][clause] = clause_value
+                    elif qp_list[i]['column'] == col and qp_list[i]['operation'] == opr:
+                        qp_list[i][clause] = clause_value
+
+    @classmethod
+    def _add_clause(cls, clause, main_key, args):
+        """
+        Adds "alias" and "format" clauses to columns in payload info. Currently, adding clauses is supported at two
+        actions only - SELECT and AGGREGATE.
+
+        :param clause: either "alias" or "format"
+        :param main_key: either "return" or "aggregate"
+        :param args: each arg is a tuple. The len of tuple depends upon main_key. If main_key is "return" i.e. SELECT,
+                     then each tuple will contain (col, alias). If main_key is "aggregate", then each tuple will contain
+                     (col, operation, alias) because col can be repeated in aggregate with different "operations".
+        :return:
+        :example:
+        PayloadBuilder().SELECT(("name", "id")).ALIAS('return', ('name', 'my_name'), ('id', 'my_id')).payload() prints
+            {"return": [{"column": "name", "alias": "my_name"},
+                        {"column": "id", "alias": "my_id"}]}
+
```

Comment:
Can we have a better docstring for this internal function? We are using the same docstring and example of ALIAS here